security/external_secrets_operator/external-secrets-monitoring.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="external-secrets-monitoring"]
+= Monitoring the External Secrets Operator for Red Hat OpenShift
+include::_attributes/common-attributes.adoc[]
+:context: external-secrets-monitoring
+
+toc::[]
+
+By default, the {external-secrets-operator} exposes metrics for the Operator and the operand. You can configure OpenShift Monitoring to collect these metrics by using the Prometheus Operator format.
+
+// Metrics scraping for external-secrets-operator
+include::modules/external-secrets-enable-operator-metrics.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../observability/monitoring/configuring-user-workload-monitoring/preparing-to-configure-the-monitoring-stack-uwm.adoc#configurable-monitoring-components_preparing-to-configure-the-monitoring-stack-uwm[Configuring user workload monitoring]
+
+// Querying metrics for the external-secrets operator
+include::modules/external-secrets-query-operator-metrics.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../observability/monitoring/accessing-metrics/accessing-metrics-as-an-administrator.adoc#accessing-metrics[Accessing metrics]
+
+
+// Enabling user workload monitoring for the external-secrets-operator operand
```

Comment:
This should be the first sub-section under `Monitoring the External Secrets Operator for Red Hat OpenShift` section.